index.cjs

```diff
@@ -782,11 +782,11 @@ safeql: try {
       ],
     },
   ];
-} catch (err) {
-  // Swallow errors in non-CI environments to avoid noisy ESLint logs in non-database projects
-  if (process.env.CI) {
-    throw err;
-  }
+} catch (error) {
+  throw new Error(
+    'The required dependencies for linting PostgreSQL have not been installed, please reinstall the ESLint config',
+    { cause: error },
+  );
```

Comment:
can you add to the PR description a screenshot of this error message in the VS Code interface?

(if the error cause doesn't get displayed or shown anywhere, we unfortunately need to do this a different way)

Reply:
I tested it locally on an e-commerce store project. For this test, I changed the `eslint-config-upleveled/index.cjs` file inside `node_modules`. The output from the VS Code terminal: 
<img width="1116" alt="Screenshot 2023-05-10 at 16 31 52" src="https://github.com/upleveled/eslint-config-upleveled/assets/80746311/44bb51de-d121-414c-8bc0-161509f2a8c2">

Reply:
the 2nd screenshot (the VS Code popup) is still missing from the PR description